code-challenges.test.js

```diff
@@ -14,19 +14,79 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const fibonacciLength1 = 6
+describe("fibonacciLength1", () => {
+  it("returns an array containing the Fibonacci sequence", () => {
+    expect(fibonacciLength1(6)).toEqual([1, 1, 2, 3, 5, 8])
+  })
+})
+
+//I broke this up into two different tests to simplify it for my own eyes. Everything was starting to blur together. 
+
+describe("fibonacciLength2", () => {
+  it("returns an array containing the Fibonacci sequence", () => {
+    expect(fibonacciLength2(10)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+  })
+})
```

Comment:
Best practice is to have 1 test for each function. Also, you are naming these functions the same as the given variables. You should be creating 1 function that will work for both variables, not separate functions for each variable. This makes your code not dynamic.